scoap3/cli_fixes.py

```diff
@@ -23,8 +27,58 @@ def init_default_location():
         loc = Location()
         loc.name = 'default'
         loc.default = True
-        loc.uri = '/virtualenv/files/'
+        loc.uri = os.environ.get('SCOAP_DEFAULT_LOCATION') or '/virtualenv/files/'
```

Comment:
This could even be added to the config. Now we don't need it anywhere else, but maybe in the future it would be nice.